### ISSUE TITLE
Newby to Pyhton

### ISSUE THREAD

**Issue by TiNivla** [Matthew-Chiang/HQSolver#1]
@Matthew-Chiang Hey so I've been looking for a HQsolver like this that runs on Windows and I finally have(Thank you!), but now close to nothing about python so I'm kinda lost on how to run this and what applications I should download to run this script.  Could you perhaps guide me in somewhat simple terms? It would be much appreciated. Thank you again.

**Comment by Matthew-Chiang** [comment]
Yeah for sure! HQSolver is still in development but I'll try to either make the readme more comprehensive or make HQSolver more user friendly.

**Comment by TiNivla** [comment]
Thanks! Much appreciated :)
